shared/p2p/hobbits/server_test.go

```diff
@@ -0,0 +1,243 @@
+package hobbits
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"reflect"
+	"testing"
+	"time"
+
+	ttl "github.com/ReneKroon/ttlcache"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/state"
+	"github.com/prysmaticlabs/prysm/beacon-chain/db"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	"github.com/renaynay/go-hobbits/encoding"
+	"github.com/renaynay/go-hobbits/tcp"
+	"gopkg.in/mgo.v2/bson"
+
```

Comment:
Remove space here, our imports are grouped in 2 groups: stdlibs and everything else.